src/api/ledger/transactions.js

```diff
@@ -62,24 +62,45 @@ function getAccountTx(remote, address, options, marker, limit, callback) {
   });
 }
 
-function getTransactionsInternal(remote, address, options, callback) {
+function checkForLedgerGaps(remote, options, transactions) {
+  let {minLedgerVersion, maxLedgerVersion} = options;
+
+  // if we reached the limit on number of transactions, then we can shrink
+  // the required ledger range to only guarantee that there are no gaps in
+  // the range of ledgers spanned by those transactions
+  if (options.limit && transactions.length === options.limit) {
```

Comment:
How often were these false positives a problem? Can you describe the issue? What are the clients for this `getTransactionsInternal` etc? 

This is used for transaction submission in any way? limits should be respected I guess when searching a particular ledger range for txns that made it into a ledger. And I guess it makes sense that if you only want say 200 of the most recent transactions, and you just put an all possible ledgers ledger range, that if you were missing some, it shouldn't really matter.  This should still detect any gaps in any of the pages though yeah?